js/templates/modals/about/donations.html

```diff
@@ -0,0 +1,51 @@
+<div class="contentBox padMd clrP clrBr clrSh3">
+  <div class="flexSpaceBetween">
+    <div class="flexRow flex0">
+      <div class="brandSmall" style="background-image: url('../imgs/baseRounded.svg');"></div>
+      <div>
+        <div><%= ob.polyT('about._brand') %></div>
+        <div class="flexRow">
+          <img src=1>
+          <img src=2>
+          <img src=3>
+        </div>
+      </div>
+    </div>
+    <h2 class="h3 clrT"><%= ob.polyT('about.donationsTab._sectionHeader') %></h2>
+    <div class="txtRight">
+      <div><%= ob.polyT('about.version') %> <%= ob.polyT('about.versionCode') %></div>
+      <a class="btn clrP clrBr js-copyAddress flexNoShrink">
+        <%= ob.polyT('about.btnUpdateCheck') %>
+      </a>
+    </div>
+  </div>
+  <hr class="clrBr" />
+  <div class="padLg">
+    <div id=donate-cta class="padMd flexBar">
+      <div id=donation-qr-code class="padMd">
+        <img src=qr>
```

Comment:
This image source looks off and results in no image loaded:

![image](https://cloud.githubusercontent.com/assets/794517/21568653/21777480-ce6b-11e6-9e9c-4211d5e0e793.png)